modules/govuk/manifests/apps/whitehall.pp

```diff
@@ -82,6 +185,16 @@
       }
       ",
     }
+
+    if $::govuk_node_class != 'development' {
+      govuk::app::envvar::database_url { $app_name:
+        type     => 'mysql',
```

Comment:
I think this needs to be mysql2 to match the name of the database adapter.  I'm getting:

```
*** [err :: whitehall-backend-1.backend.integration.publishing.service.gov.uk] Specified 'mysql' for database adapter, but the gem is not loaded. Add `gem 'mysql'` to your Gemfile (and ensure its version is at the minimum required by ActiveRecord).
```

When deploying and running the migrations.